### PR TITLE
normalized-vendor-and-records

### PR DESCRIPTION
**Pull request by shaikegross**
- Created NormalizedVendor type
- Created MatchRecord type
- Regenerated clients
 - Changed bad setting in eslintrc

### CODE REVIEW

File: src/codegen/client/model/formatResponse.ts [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+        {
+            "name": "formatError",
+            "baseName": "formatError",
+            "type": "ErrorResponse"
+        }    ];
+
+    static getAttributeTypeMap() {
+        return FormatResponse.attributeTypeMap;
+    }
+}
```
Comment (yarivkohn) [inline src/codegen/client/model/formatResponse.ts:50]: 1. You have the same functions and var in 4 different files. 
Why don't you make the inherit from a parent class?
2. The array type of AttributeTypeMap is repetitive as well, I think you can create a specific type for it

File: src/handlers/match-handler/index.ts [hunk 2]
```diff
@@ -1,3 +1,41 @@
...
+    });
+  } catch (err) {
+    console.error({ err });
+
+    return {
+      body: err,
+      statusCode: 500,
+      headers: undefined,
+      isBase64Encoded: false,
+      multiValueHeaders: undefined,
```
Comment (yarivkohn) [inline src/handlers/match-handler/index.ts:36]: I think we need to create an object for success and failure responses

**Comment by shaikegross** [comment]
> I think we need to create an object for success and failure responses 

Yea agreed, this is just test code I used to run locally and see that I can return a response. will convert to sensible objects

File: src/codegen/client/model/formatResponse.ts (hunk 1 above)
Comment (shaikegross) [reply src/codegen/client/model/formatResponse.ts:50]: This is auto generated code. It's being generated by the openapi-generator

File: src/handlers/match-handler/index.ts (hunk 2 above)
Comment (shaikegross) [reply src/handlers/match-handler/index.ts:36]: Yea agreed, this is just test code I used to run locally and see that I can return a response. will convert to sensible objects

**Review by shaulzorea — APPROVED** [review]

**Pull request merged**